cv32/sim/Common.mk

```diff
@@ -83,18 +86,27 @@ COMPLIANCE_BRANCH ?= master
 # 2020-08-19
 COMPLIANCE_HASH   ?= c21a2e86afa3f7d4292a2dd26b759f3f29cde497
 
-# Generate command to clone the CV32E40P RTL
+# Generate command to clone the CV32E40P RTL.
+# If a TAG is specified, get the master branch from that tag
+
 ifeq ($(CV32E40P_BRANCH), master)
-  TMP = git clone $(CV32E40P_REPO) --recurse $(CV32E40P_PKG)
+  TMP = git clone $(CV32E40P_REPO) $(CV32E40P_PKG)
 else
-  TMP = git clone -b $(CV32E40P_BRANCH) --single-branch $(CV32E40P_REPO) --recurse $(CV32E40P_PKG)
+  TMP = git clone -b $(CV32E40P_BRANCH) --single-branch $(CV32E40P_REPO) $(CV32E40P_PKG)
 endif
 
 ifeq ($(CV32E40P_HASH), head)
   CLONE_CV32E40P_CMD = $(TMP)
 else
   CLONE_CV32E40P_CMD = $(TMP); cd $(CV32E40P_PKG); git checkout $(CV32E40P_HASH)
 endif
+
```

Comment:
Would like for this code to be in one ifeq endif section.  I believe the code as written could cause a race condition as Make tries to solve CLONE_CV32E40P_CMD as make tries to determine the proper resolution order of variables.

Reply:
Good point.  Pushed in a change that does it with nested ifeq/endif sections.